docs/dg/dev/architecture/coding-conventions.md

```diff
@@ -0,0 +1,2114 @@
+{% info_block infoBox "Info" %}
+
+This document serves as a dynamic resource intended to provide an overview of our conventions and guidelines. We acknowledge that the technology landscape is ever-evolving, and as such, this document is subject to continuous refinement and improvement.
+
+Your feedback and suggestions are highly valued to enhance the accuracy, relevance, and effectiveness of Spryker Commerce Operating System (SCOS). We encourage you to contribute your insights and recommendations by proposing changes through our designated channels.
+
+{% endinfo_block %}
+
+# Documentation structure
+
+## Development Use-Cases
```

Comment:
Add more clarity on the convention structure to the readers to make it simpler to understand what conventions apply for a given case.